packages/components/src/components/post-megadropdown/post-megadropdown.tsx

```diff
@@ -214,6 +244,27 @@ export class PostMegadropdown {
     }
   }
 
+  private handleAriaCurrentChange(mutations: MutationRecord[]) {
```

Comment:
This method is doing almost the same thing as `checkInitialAriaCurrent`. You can refactor the code to avoid duplicated logic, for example with a `setTriggerActive(isActive: boolean)` function.